### PR TITLE
[FE] Public results: open modal right away creating filter set

### PR DESCRIPTION
**Pull request by shirve**
### Changes
Improved filter set modal to open right away on filter create

### CODE REVIEW

**Comment by tobob** [comment]
@shirve your implementation is not looking good, it should work only when the user create a filter
now if the user opens results for the company/team (they are persisted on BE) it also automatically opens the modal when entering the results screen


https://user-images.githubusercontent.com/3189332/201093110-f4aba2d0-2550-44d0-9da9-c6bab7338c08.mov

**Review by tobob — CHANGES_REQUESTED** [review]
invalid implementation, please check company/team results view

File: team-fingerprints-web/src/components/FiltersSets/FiltersSets.tsx [hunk 1]
```diff
@@ -25,6 +25,8 @@ const FiltersSets = ({
   apiUrl,
   isPublic,
 }: Props) => {
+  const [newestFilterSet, setNewestFilterSet] = useState<FiltersSet>();
```
Comment (tobob) [inline team-fingerprints-web/src/components/FiltersSets/FiltersSets.tsx:28]: @shirve please don't save whole filter set object, rather please save only _id of filterset

**Review by tobob — CHANGES_REQUESTED** [review]
please don't save whole object, only _id of it

File: team-fingerprints-web/src/components/FiltersSets/FiltersSets.tsx (hunk 1 above)
Comment (shirve) [reply team-fingerprints-web/src/components/FiltersSets/FiltersSets.tsx]: Applied

**Review by tobob — APPROVED** [review]

**Pull request merged**